src/domain/callout/creation-dialog/steps/CalloutSummaryStep/CalloutSummaryStep.tsx

```diff
@@ -53,7 +50,7 @@ export default CalloutSummaryStep;
 
 const CalloutSummary: FC<{
   callout: CalloutDialogCreationType;
-  templatePreviewComponent: ComponentType<CalloutStepProps> | null;
+  templatePreviewComponent?: ComponentType<CalloutStepProps> | null;
```

Comment:
you already can pass a null here, I think with the question mark is enough

Reply:
I see now the logic behind that  `| null` ...
Simplify it if you can, if now just resolve this comment, this is temporary code anyway... 
Approving

Reply:
I don't see how. Can you suggest something?